TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+# CesiumJS and Webpack
+
+[Webpack](https://webpack.js.org/) is a popular and powerful tool for bundling JavaScript modules. It allows developers to structure their code and assets in an intuitive way and to load different kinds of files as needed with simple require statements. When building, it will trace code dependencies and pack these modules into one or more bundles that are loaded by the web browser.
+
+In this tutorial, we will build a simple web app from the ground up using webpack, and then cover the steps to integrate the [Cesium npm module](https://www.npmjs.com/package/cesium). This is a good place to start if you’d like to use CesiumJS to develop a web application. If you’re new to Cesium and are looking to learn to build your first sample app, take a look at our [Getting Started Tutorial](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/).
```

Comment:
Our tutorials language always addresses the audience. For example "In this tutorial, you will build...". See our website tutorials for more.

Please take a pass through the full document for these changes.

Reply:
I just added a commit that addresses this feedback. I originally used "we" in various places because I wanted the tutorial reader to feel welcomed - almost as if I was working with them to get webpack running in their development environment.